development/tools/cmd/usersmapchecker/main.go

```diff
@@ -97,30 +103,46 @@ func main() {
 			contextLogger.LogError(fmt.Sprintf("error when getting pr author for presubmit: got error %v", err))
 		}
 	}
+	org, err := prow.GetOrgForPresubmit()
+	if err != nil {
+		if notPresubmit := prow.IsNotPresubmitError(err); *notPresubmit {
+			contextLogger.LogInfo(err.Error())
+		} else {
+			contextLogger.LogError(fmt.Sprintf("error when getting org for presubmit: got error %v", err))
+		}
+	}
 	// TODO: move searching of user in to kymabot package
 	wg.Add(len(authors))
 	contextLogger.LogInfo(fmt.Sprintf("found %d authors in job spec env variable", len(authors)))
 	// Search entries for authors github usernames.
 	for _, author := range authors {
-		// Use goroutines.
-		go func(wg *sync.WaitGroup, author string, exitCode *atomic.Value) {
-			// Notify goroutine is done when exiting from it.
-			defer wg.Done()
-			for _, user := range usersMap {
-				if user.ComGithubUsername == author {
-					contextLogger.LogInfo(fmt.Sprintf("user %s is present in users map", author))
-					return
+		member, _, err := githubComClient.Organizations.IsMember(ctx, org, author)
```

Comment:
if `err` is unused it can be removed here

Reply:
:) It should be used. Will add some body to the if block.

Reply:
@Halamix2 Added missing error handling block.